lib/api/one-span.js

```diff
@@ -1,6 +1,12 @@
 module.exports = {
   PACKAGES: {
     ROUTE: "/packages",
+    STATUS: {
+      draft: "DRAFT",
```

Comment:
This looks good. Only thing I would change is capitalizing these keys, just so it's consistent.

Reply:
Got it 👍